docs/src/theme/DocSidebarItems/index.js

```diff
@@ -5,7 +5,7 @@ import SidebarLabel from '@site/src/components/SidebarLabel';
 import styles from './styles.module.css';
 
 const EXPERIMENTAL_APIs = ['shared-element-transitions/overview'];
-const NEW_APIS = ['animations/withClamp'];
+const NEW_APIS = ['animations/withClamp', 'advanced/useComposedEventHandler'];
```

Comment:
Should we drop `withClamp` as new? cc @tomekzaw

Reply:
Yeah let's just remove the label for `withClamp`